### ISSUE TITLE
The story of how an Ottawa company ported React Native to 10 platforms

### ISSUE THREAD

**Issue by DCLAN** [OttawaReactJS/talks#11]
I'd like to talk about the crazy things we're doing with React Native at youi.tv!
We've been a C++ company for quite some time, and we have a cross platform youi rendering to 10 different platforms (mobile, and game consoles, streaming sticks).

But I don't know if you heard, but writing UIs with C++ isn't exactly all the rage these days, so we ported React Native to our SDK, and amazingly it worked. We now have multiple customers using React Native and we've shipped the only application on React Native for Roku.

We'll discuss and showcase how this all works, and what we still have to work on as well.

**Comment by btav** [comment]
I wouuuuullddddd loooooooooooooooove to hear this talk! Are you interested in speaking this month? @DCLAN

**Comment by DCLAN** [comment]
Would love to, but I may be travelling next week - when is the date again? @btav

**Comment by DCLAN** [comment]
@matldupont pointed out to me it's on the 27th... I first need to confirm if I'm travelling. I'll try to find a potential backup as well. 

TLDR: Let me get back to you

**Comment by btav** [comment]
This talk was awesome!
Closing old issue!
